src/docfx/build/link/LinkResolver.cs

```diff
@@ -156,7 +155,13 @@ internal class LinkResolver
                 return (errors, "", fragment, LinkType.AbsolutePath, null, false);
```

Comment:
It is still possible for an image to have host name like `docs.microsoft.com`.

Reply:
Not sure, but `docs.microsoft.com` should be a trusted domain, and it will not enter the `if` statement (line 146)